apps/web/src/api/subscribers.ts

```diff
@@ -0,0 +1,5 @@
+import { api } from './api.client';
+
+export function getSubscribersList(page = 0, limit = 10) {
```

Comment:
this function should be typed, rn the return type is `Promise<any>`

Reply:
Yes, you are right
But I see every function is missing type here
https://github.com/novuhq/novu/tree/main/apps/web/src/api